docs/tosa-op-coverage.md

```diff
@@ -20,5 +20,6 @@ The table below shows the supported TOSA ops.
 | mul                   | :heavy_check_mark: | |
 | sub                   | :heavy_check_mark: | |
 | **Other ops**
+| conv2d                | :white_check_mark: | Quantization not supported |
```

Comment:
Not only quantization but also dilated convs are not supported.

Reply:
I updated the doc :+1: